src/core.js

```diff
@@ -212,7 +212,10 @@ jQuery.extend( {
 	noop: function() {},
 
 	isFunction: function( obj ) {
-		return jQuery.type( obj ) === "function";
+
+		// Support: Chrome <=57, Firefox <=52
+		// Don't classify callable <object>s as functions
+		return typeof obj === "function" && typeof obj.nodeType !== "number";
```

Comment:
"Don't classify callable \<object> elements as functions` is perfectly clear to me. Can you just tell me what you want to see here?